### PR TITLE
enhance Java/11 wrapper to also support for aarch64/Arm

### LINKED ISSUE #11184: Java.11.eb lacks support for arm/aarch64.
```
ERROR: Failed to process easyconfig /home/terjekv/easybuild/software/EasyBuild/4.2.2/easybuild/easyconfigs/j/Java/Java-11.eb: No matches for version in {'arch=x86_64': '%(version)s.0.2', 'arch=POWER': '%(version)s.0.6-ppc64le'} (looking for arch=AArch64)
```

### PR DESCRIPTION
**Pull request by terjekv**
(created using `eb --new-pr`)

fixed #11184

### CODE REVIEW

File: easybuild/easyconfigs/j/Java/Java-11.0.8-aarch64.eb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+homepage = 'http://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/download/jdk-%(version)s%2B10/']
+sources = ['OpenJDK11U-jdk_%(arch)s_linux_%(version)s_10.tar.gz']
+# source_urls = ['https://download.java.net/java/GA/jdk%(version_major)s/9/GPL/']
```
Comment (Flamefire) [inline easybuild/easyconfigs/j/Java/Java-11.0.8-aarch64.eb:13]: Please remove the commented line and add a checksum

Also: Is the `%(arch)s` template really useful given this is already specific to aarch64? And isn't `arch=AArch64`? The filename needs that lowercased. Do we have `arch_lower`?

**Review by Flamefire — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/Java/Java-11.0.8-aarch64.eb (hunk 1 above)
Comment (terjekv) [reply easybuild/easyconfigs/j/Java/Java-11.0.8-aarch64.eb]: Thanks! Fixes incoming.

**Comment by boegel** [comment]
@terjekv Any chance for a test report from an Arm system?

**Comment by terjekv** [comment]
I'll have to wrangle it a bit. Should be doable.

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 0 out of 0 (2 easyconfigs in this PR)
arm1 - Linux ubuntu 18.04, AArch64, UNKNOWN (aarch64), Python 3.6.9
See https://gist.github.com/2f398e75f23fac55c109655d7b8b73d5 for a full test report.

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
arm1 - Linux ubuntu 18.04, AArch64, UNKNOWN (aarch64), Python 3.6.9
See https://gist.github.com/8b70c629c25e86b4d4ef2877c78dcc52 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @terjekv!

**Pull request merged**